### ISSUE TITLE
A11y Semantics for bottom bar in single and multi-screen sims

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#94]
See https://github.com/phetsims/a11y-research/issues/25 for historical discussion.

In March 6th Keyboard Nav Meeting, we decided to re-structure the bottom bar to better communicate the function/role of the bottom bar and to better communicate access to sim screen buttons in multi-screen sims.

@terracoda will post new HTML here and we can discuss any further code or design changes here.

**Comment by terracoda** [comment]
**Using Build and Atom as an Example:**
- [Home Screen](http://htmlpreview.github.io/?https://github.com/phetsims/a11y-research/blob/master/html-sketches/sim-screens/index.html)
- [Atom Screen](http://htmlpreview.github.io/?https://github.com/phetsims/a11y-research/blob/master/html-sketches/sim-screens/baa-01-atom-template.html)

**Comment by terracoda** [comment]
I updated the actual template files.
Will add 2 more examples to see what semantics we hear on the phet menu items.

**Comment by terracoda** [comment]
Adding links to the template files:
**Using Build and Atom as an Example:**
- [Home Screen Template](https://htmlpreview.github.io/?https://github.com/phetsims/a11y-research/blob/master/html-sketches/sim-screens/template-sim-screen/00-home-screen.html)
- [Atom Screen](https://htmlpreview.github.io/?https://github.com/phetsims/a11y-research/blob/master/html-sketches/sim-screens/template-sim-screen/01-screen-template.html

**Comment by terracoda** [comment]
I have created a Design Pattern for multi-screen sims in joist. I used code from these mock-ups buyt have iterated a little bit.
I will update these mock-ups once the design patterns is ok'd by the team.

**Comment by terracoda** [comment]
Mark down file for the Home Screen is in [joist/doc/HomeScreen.md](https://github.com/phetsims/joist/blob/master/doc/HomeScreen.md)

**Comment by terracoda** [comment]
@zepumph, I have updated the index.html file in the baa-screens folder. I have put all the sim content on one page, so everything can be seen together.

FYI, I put horizontal rules in between sim screen.

- [Multi-screen Sim Example](https://htmlpreview.github.io/?https://github.com/phetsims/a11y-research/blob/master/html-sketches/sim-screens/baa-screens/index.html)

Note that the code validates with warnings about the multiple H1 elements, and thee JS that is supposed to collapse and expand the PhET Menu is not quite right. I'm not sure why it works for one button, but not the other.

**Comment by terracoda** [comment]
When on git hub, the javascript for the PhET Menu does not work on either PhET Menu button.

**Comment by terracoda** [comment]
The example in https://github.com/phetsims/a11y-research/issues/94#issuecomment-398888379 is updated, and awaiting review by team.

**Comment by terracoda** [comment]
Reminder, that I will need to add the Global Mute Sound button at some point before September.

**Comment by zepumph** [comment]
This issue is quite stale, likely the bottom bar is looking really good for single screen sims, since we have published a few already. Unassigning myself until work continues for multi-screen sims and my feedback and time is needed.

**Comment by terracoda** [comment]
I will double check what has been implemented and see if anything is missing. True, the bottom with mute sound, keyboard shortcuts, and PhET menu has been working fine for people thus far.

**Comment by terracoda** [comment]
We have design the multi-screen sim for Ratio and Proportion.

We can use the sim as a good pattern. The buttons were updated to work well with desktop screen readers and mobile VoiceOver.

Here are screen shots of the final Home Screen, Discover Screen and Create Screen for Ration and Proportion.
The A11y View text shows that when changing screens, focus starts at the top. On the Home Screen, I just happen to have my focus on the first screen button.

<img width="1200" alt="Screen Shot 2021-02-03 at 3 17 31 PM" src="https://user-images.githubusercontent.com/1902116/106794441-6ba0d600-6633-11eb-880b-4e88c5878beb.png">
<img width="1201" alt="Screen Shot 2021-02-03 at 3 17 50 PM" src="https://user-images.githubusercontent.com/1902116/106794446-6e033000-6633-11eb-9fb7-cf6aaec9b4ad.png">
<img width="1195" alt="Screen Shot 2021-02-03 at 3 18 09 PM" src="https://user-images.githubusercontent.com/1902116/106794452-6fccf380-6633-11eb-82e0-b0b176a783c5.png">

**Comment by terracoda** [comment]
Oh this might not be the right issue for the previous comment.

**Comment by terracoda** [comment]
Moving previous comment it #104

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/a11y-research/issues/94#issuecomment-772737948 look great! This issue was also about the navigation bar, is there anything else to do for that or can this issue be closed now?

**Comment by terracoda** [comment]
Ad just an FYI, I updated https://github.com/phetsims/joist/blob/master/doc/HomeScreen.md to include Ratio and Proportion as one of the examples.
